### PR TITLE
[WEBSITE-2165] Dynamically generate title depending on House

### PR DESCRIPTION
**Pull request by jennaramdenee**
* Amended views, adding logic to show 'MPs' if House of Commons, and 'Lords' if House of Lords

### CODE REVIEW

File: app/views/parliaments/houses/members/a_to_z.html.haml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-- title("#{t('mp_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')}")
+- title("#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')}")
 #content.section--primary{ tabindex: "0" }
   .container
     %h1
-      %span= "#{t('mp_plural')}"
-      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament').capitalize}", parliament_path(@parliament.graph_id))
+      %span= "#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')}"
+      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament')}", parliament_path(@parliament.graph_id))
```
Comment (mattrayner) [inline app/views/parliaments/houses/members/a_to_z.html.haml]: Do we still need to capitalise Parliament?

File: app/views/parliaments/houses/members/index.html.haml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-- title("#{t('mp_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')}")
+- title("#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')} - #{@parliament.date_range(date_format: '%Y')} #{t('parliament')}")
 .section--primary
   .container
     %h1
-      %span= t('mp_plural')
-      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament').capitalize}", parliament_path(@parliament.graph_id))
+      %span= "#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')}"
+      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament')}", parliament_path(@parliament.graph_id))
```
Comment (mattrayner) [inline app/views/parliaments/houses/members/index.html.haml]: same as above

File: app/views/parliaments/houses/members/letters.html.haml [hunk 3]
```diff
@@ -1,9 +1,9 @@
-- title("#{t('mp_plural')} #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')} #{t('parliaments.showing_results')} '#{params[:letter].capitalize}'")
+- title("#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')} #{@parliament.date_range(date_format: '%Y')} #{t('parliament')} #{t('parliaments.a_to_z')} #{t('parliaments.showing_results')} '#{params[:letter].capitalize}'")
 .section--primary
   .container
     %h1
-      %span= t('mp_plural')
-      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament').capitalize}", parliament_path(@parliament.graph_id))
+      %span= "#{Parliament::Utils::Helpers::HousesHelper.commons?(@house) ? t('mp_plural') : t('lord_plural')}"
+      = link_to("#{@parliament.date_range(date_format: '%Y')} #{t('parliament')}", parliament_path(@parliament.graph_id))
```
Comment (mattrayner) [inline app/views/parliaments/houses/members/letters.html.haml]: same

File: app/views/parliaments/houses/members/a_to_z.html.haml (hunk 1 above)
Comment (jennaramdenee) [reply app/views/parliaments/houses/members/a_to_z.html.haml]: Reviewed with Alan, who would like 'parliament' to stay lowercase across the whole site

**Pull request merged**